packages/amplify-provider-awscloudformation/src/push-resources.js

```diff
@@ -511,6 +511,12 @@ function formNestedStack(context, projectDetails, categoryName, resourceName, se
   /* eslint-enable */
   const initTemplateFilePath = path.join(__dirname, '..', 'resources', 'rootStackTemplate.json');
   const nestedStack = context.amplify.readJsonFile(initTemplateFilePath);
+
+  // Track Amplify Console generated stacks
+  if (process.env.CLI_DEV_INTERNAL_DISABLE_AMPLIFY_APP_DELETION) {
+    nestedStack.Description = 'Root stack created from AWS Amplify Console';
```

Comment:
Root Stack for AWS Amplify Console

Reply:
IMHO it is not correct as it is not "for" Amplify Console. The stack is "for" the project, created by/with CLI or Console seems to be right.

But my bigger concern here is why not define and use an "amplify:creator" tag to track this? depending on description does not seems to be ideal...